docs/reference/expectations/result_format.md

```diff
@@ -419,28 +378,20 @@ The complete `result` includes:
     'success': False,
     'result': {
         'observed_value': The aggregate statistic computed for the column (also in `SUMMARY`)
-        'element_count': The total number of values in the column (also in `SUMMARY`)
-        'missing_count':  The number of missing values in the column (also in `SUMMARY`)
-        'missing_percent': The total percent of missing values in the column (also in `SUMMARY`)
```

Comment:
Good catch. Just to verify - these don't appear for all expectations? Any idea why they were included here (i.e. were they available at one point but not any longer and we didn't update the docs)?